### ISSUE TITLE
Crash when loading terrain with missing page file

### ISSUE THREAD

**Issue by Michael10055** [RigsOfRods/rigs-of-rods#1437]
If a terrain is missing its page file (example: [N-Labs](https://archives.rigsofrods.org/repo/0.4-Terrains/CM%200.4.zip)) RoR will crash.
```
11:05:51:  ===== LOADING TERRAIN GEOMETRY CM.terrn2
11:05:51:   ## Loading Terrain Geometry
11:05:51: OGRE EXCEPTION(6:FileNotFoundException): Cannot locate resource CM-page-0-0.otc in resource group General or any other group. in ResourceGroupManager::openResource at E:\RoR-data\ror-dependencies\build\Source\ogre\OgreMain\src\OgreResourceGroupManager.cpp (line 756)
```
Terrain loads fine after adding the file (even if it is empty).

**Comment by tritonas00** [comment]
same here

**Comment by tritonas00** [comment]
shameless bump :smiley:

**Comment by only-a-ptr** [comment]
Eh, allright then 😄

**Comment by tritonas00** [comment]
shameless bump (version 2 :smile:)

**Comment by only-a-ptr** [comment]
For whatever reason the fix didn't make it to upstream... I'll revisit it.

**Comment by only-a-ptr** [comment]
Uhh, the terrn2 format is so under-designed. Anyway, after thoroughly examining the problem, I concluded:

* [the docs](http://docs.rigsofrods.org/terrain-creation/terrn2-subsystem/#ogre-terrain-page-config-otc) say that there must be a `page-X-Z.otc` file for every page, so let's leave it that way. 
* The workaround using empty `page*.otc` file is actually a bug, but I'll keep it for compatibility and comment it as such in the code.
* If the `page*.otc`  file is missing, RoR will return to menu and display error.
* The N-Labs map was updated to use the workaround, get it here: https://forum.rigsofrods.org/downloads.php?do=file&id=228
* If the modder wishes to disable loading heightmap, he can specify `Flat=true` in terrn2 file.
* Disabling the terrain geometry altogether is not supported and I'm OK with that.
